spec/requests/posts_spec.rb

```diff
@@ -2,6 +2,46 @@
 
 RSpec.describe 'Posts', type: :request do
```

Comment:
- [ ] @emmanuellmhango great work on your tests, you covered all areas requested however am getting the error below when running your tests, this is caused by code in posts_helper_spec.rb and users_helper_spec.rb line 14
![image](https://user-images.githubusercontent.com/61408860/223550090-c9e8407d-1da9-4044-bc8a-da0114597cc0.png)
, you are suppose to add an example instead of leaving it as it is,
![image](https://user-images.githubusercontent.com/61408860/223549359-cd8230e2-6442-428d-b8ff-c71fe9cee574.png)
![image](https://user-images.githubusercontent.com/61408860/223549426-682e0627-6d01-42b2-9b3e-6fa53e83851f.png)

Reply:
dear @danmainah 
Thank you for your comment. all tests have passed from my side. may you assist as to what such errors can be resolved?